tests/contrib/pymongo/test_pymongo_patch.py

```diff
@@ -2,13 +2,13 @@
 # script. If you want to make changes to it, you should make sure that you have
 # removed the ``_generated`` suffix from the file name, to prevent the content
 # from being overwritten by future re-generations.
-
-from ddtrace.contrib.pymongo import get_version
-from ddtrace.contrib.pymongo.patch import patch
+from ddtrace.contrib.internal.pymongo.patch import _VERSION
+from ddtrace.contrib.internal.pymongo.patch import get_version
+from ddtrace.contrib.internal.pymongo.patch import patch
 
 
 try:
-    from ddtrace.contrib.pymongo.patch import unpatch
+    from ddtrace.contrib.internal.pymongo.patch import unpatch
```

Comment:
I don't think we need this try-except. The unpatch function should always exist 

```suggestion
from ddtrace.contrib.pymongo import unpatch
```